src/carebot-tracker.js

```diff
@@ -304,23 +313,35 @@
             var percent = depthPercent();
             var bucket = percentBucket(percent);
             if (bucket > previousBucket) {
-                var seconds = timer.check().seconds;
-                callback(bucket, seconds);
-            } else {
-                // The user is scrolling back up.
+                callback(bucket, timer.check().seconds);
                 previousBucket = bucket;
```

Comment:
Hi @ahilles107 -- just wanted to check in and see if you're good with pulling out the `previousBucket = bucket` line. We'd like to deploy the improvements!